### ISSUE TITLE
Why don't we commit package-lock.json?

### ISSUE THREAD

**Issue by bl-ue** [tldr-pages/tldr-lint#40]
Why don't we commit `package-log.json` as almost all Node.js/npm projects universally do?

```console
    $ npm install
    npm WARN deprecated nomnom@1.5.2: Package no longer supported. Contact support@npmjs.com for more info.
    npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
--> npm notice created a lockfile as package-lock.json. You should commit this file.
    npm WARN notsup Unsupported engine for jasmine@3.6.3: wanted: {"node":"^10 || ^12 || ^14"} (current: {"node":"15.4.0","npm":"6.14.8"})
    npm WARN notsup Not compatible with your version of node/npm: jasmine@3.6.3

    added 130 packages from 142 contributors and audited 130 packages in 3.359s

    7 packages are looking for funding
      run `npm fund` for details

    found 0 vulnerabilities
```

Quoting from https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json:
>This file is intended to be committed into source repositories, and serves various purposes:
>
>- Describe a single representation of a dependency tree such that teammates, deployments, and continuous integration are guaranteed to install exactly the same dependencies.
>
>- Provide a facility for users to "time-travel" to previous states of node_modules without having to commit the directory itself.
>
>- To facilitate greater visibility of tree changes through readable source control diffs.
>
>- And optimize the installation process by allowing npm to skip repeated metadata resolutions for previously-installed packages.

**Comment by owenvoke** [comment]
I don't see the point for this as it's a library, and the lockfile won't be used when actually using the linter in a project.

**Comment by bl-ue** [comment]
That's not conventional...but if you think so, I do to.

**Comment by owenvoke** [comment]
I'm completely happy with committing a lockfile. 👍🏻 Personally I just tend to not for libraries that are used by end users, however we could just include it and Dependabot should handle it.

**Comment by bl-ue** [comment]
I'll wait for #38 to be merged just to avoid complexities.

**Comment by owenvoke** [comment]
@bl-ue, I've merged #38 so this is fine to do. 👍🏻 Cheers.

**Comment by bl-ue** [comment]
@owenvoke Done.  #41
